README.rst

```diff
@@ -1,5 +1,38 @@
 HCA DSS CLI
 ===========
+This repository contains a prototype for interacting with the replicated data storage system (aka the "blue box") of the Human Cell Atlas.

```

Comment:
In my personal opinion, we should avoid using the "blue/green/red box" terminology in formal documentation. Feel free to push back though.